_posts/2016/2016-05-31-gradle-jenkins-plugin.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "How to empower your Jenkins with Gradle"
+date: "2016-05-31 10:59:11"
```

Comment:
Could you have a look here https://www.pexels.com/ or https://pixabay.com/ for a picture, that matches your post, thanks. We can also do it together, if that helps.

You would have to include it like that below _date_:
image: blog-header/xxx.jpg